FMI-CROSS-CHECK-RULES.md

```diff
@@ -131,14 +103,16 @@ Vendors are encouraged to produce FMUs that are also free of all FMI Compliance
 
 FMUs for the “c-code” platform are exempt from this rule until the FMI Compliance Checker supports the “c-code” platform.
 
+// TODO: what about FMPy here? Second alternative? Drop this entirely?
```

Comment:
It would be great to add the C-Code FMUs to the Cross-check. @t-sommer : is this possible with fmpy?

Reply:
Yes, FMPy can do that for a while now, this is "just another platform".